test/e2e/adminUI/pages/fieldTypes/url.js

```diff
@@ -0,0 +1,39 @@
+var utils = require('../../../utils');
+
+module.exports = function UrlType(config) {
+	var self = {
+		selector: '.field-type-url[for="' + config.fieldName + '"]',
+		elements: {
+			label: '.FormLabel',
+			value: 'input[name="' + config.fieldName + '"]',
+		},
+		commands: [{
```

Comment:
I'll make another PR without the merge, and will everything in one commit. I'll also change the entries to be URLs, so that the test passes if the issue is fixed. Presumably when it's fixed it's a good idea to check a non-url throws an error.